### ISSUE TITLE
Unable to add parameters on "/all" endpoint

### ISSUE THREAD

**Issue by boston2029** [RazorSh4rk/random-word-api#23]
If I try to make a request (just going into the URL in my browser btw.) it won't accept params. Example URL: `https://random-word-api.herokuapp.com/all?length=5`

Example output:

```
Unknown argument: "length"

Arguments provided did not match expected signature:

getAll
  req  cask.Request
  lang  String (default en)

```

**Comment by RazorSh4rk** [comment]
Yes, the /all endpoint just returns all words. If you want to filter, use a set number, but letting it run on the entire set on a heroku dyno would just hog the app. If you want a lot of words, you can `/word?count=9999&length=5`
